### DISCUSSION TITLE
Live transcription in Android - Kotlin

### DISCUSSION THREAD

**Discussion by i-sachinkumar** [deepgram/community#1016] [Feature Requests]
I want to use Deepgram API to transcribe streaming audio input through android's microphone. Either Java or Kotlin supported SDK is required. Can you please assist when and how can I get that

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- A request ID that triggered your error or issue.

**Reply by jcdyer** [reply]
It's true we don't currently have SDKs for any of the JRE-based languages. However, Deepgram's core offering is a standard REST API, so an SDK is not required.  We have a few [code examples in Java](https://github.com/deepgram-devs/code-samples/tree/main/languages/java), as well as [more examples in several other languages](https://github.com/deepgram-devs/code-samples/tree/main/languages).  Hopefully that will get you pointed in the right direction.
